diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/indentation/Checkers.kt

```diff
@@ -84,8 +96,15 @@ internal class ValueParameterListChecker(configuration: IndentationConfig) : Cus
                     }
 
             val expectedIndent = if (parameterAfterLpar != null && configuration.alignedParameters && parameterList.elementType == VALUE_PARAMETER_LIST) {
-                // fixme: probably there is a better way to find column number
-                parameterList.parents().last().text.substringBefore(parameterAfterLpar.text).lines().last().count()
+                // count column number of the first parameter
+                val ktFile = whiteSpace.parents.last() as KtFile
+                ktFile.text
+                        .lineSequence()
+                        .scan(0 to "") { (length, _), s -> length + s.length + 1 to s }
```

Comment:
please add comments of what are you doing here or move to some method. Now it looks like some hack, I do not understand

Reply:
Yes, I will add comments. Basically, I just find a particular line using node's offset and then calculate column number for this node.